addon/components/o-s-s/iconName.enum.ts

```diff
@@ -103,7 +103,7 @@ export enum IconNames {
   TrashAlt = 'fa-trash-alt',
   Truck = 'fa-truck',
   Twitch = 'fa-twitch',
-  Twitter = 'fa-twitter',
+  Twitter = 'fa-x-twitter',
```

Comment:
not sure about this one, i think Owen did it so we can type icon usages at some point. Meaning Twitter is untouched & it's more like you need to add an: `XTwitter = 'fa-x-twitter` option.